src/yuzu/main.cpp

```diff
@@ -2840,170 +2840,335 @@ void GMainWindow::OnGameListNavigateToGamedbEntry(u64 program_id,
+
+            fmt::print(shortcut_stream, "[Desktop Entry]\n");
+            fmt::print(shortcut_stream, "Type=Application\n");
+            fmt::print(shortcut_stream, "Version=1.0\n");
+            fmt::print(shortcut_stream, "Name={}\n", name);
+
+            if (!comment.empty()) {
+                fmt::print(shortcut_stream, "Comment={}\n", comment);
+            }
+
+            if (std::filesystem::is_regular_file(icon_path)) {
+                fmt::print(shortcut_stream, "Icon={}\n", icon_path.string());
+            }
+
+            fmt::print(shortcut_stream, "TryExec={}\n", command.string());
+            fmt::print(shortcut_stream, "Exec={}", command.string());
+
+            if (!arguments.empty()) {
+                fmt::print(shortcut_stream, " {}", arguments);
+            }
+
+            fmt::print(shortcut_stream, "\n");
+
+            if (!categories.empty()) {
+                fmt::print(shortcut_stream, "Categories={}\n", categories);
+            }
+
+            if (!keywords.empty()) {
+                fmt::print(shortcut_stream, "Keywords={}\n", keywords);
+            }
+
+            // Flush and close file
+            shortcut_stream.flush();
+            shortcut_stream.close();
+            return true;
+        } else {
+            LOG_ERROR(Frontend, "Failed to create shortcut");
+        }
+        shortcut_stream.close();
+    } catch (const std::exception& e) {
+        LOG_ERROR(Frontend, "Failed to create shortcut: {}", e.what());
     }
-    case GameListShortcutTarget::Applications: {
-        const QString applications_path =
-            QStandardPaths::writableLocation(QStandardPaths::ApplicationsLocation);
-        if (applications_path.isEmpty()) {
-            const char* home = std::getenv("HOME");
-            if (home != nullptr) {
-                target_directory = std::filesystem::path(home) / ".local/share/applications";
+    return false;
+#elif defined(_WIN32) // Windows
+    HRESULT hr = CoInitialize(NULL);
+    if (SUCCEEDED(hr)) {
+
+        IShellLinkW* ps1 = nullptr;
+        IPersistFile* persist_file = nullptr;
+
+        SCOPE_EXIT({
+            if (persist_file != nullptr) {
+                persist_file->Release();
+            }
+            if (ps1 != nullptr) {
+                ps1->Release();
+            }
+
+            CoUninitialize();
+        });
+
+        HRESULT hres = CoCreateInstance(CLSID_ShellLink, NULL, CLSCTX_INPROC_SERVER,
+                                        IID_IShellLinkW, (void**)&ps1);
+
+        std::wstring wshortcut_path_full =
+            Common::UTF8ToUTF16W((shortcut_path_full).string() + ".lnk");
+        std::wstring wicon_path = Common::UTF8ToUTF16W(icon_path.string());
+        std::wstring wcommand = Common::UTF8ToUTF16W(command.string());
+        std::wstring warguments = Common::UTF8ToUTF16W(arguments);
+        std::wstring wcomment = Common::UTF8ToUTF16W(comment);
+
+        if (SUCCEEDED(hres)) {
```

Comment:
```suggestion
        if (!SUCCEEDED(hres)) {
            LOG_ERROR(Frontend, "Failed to create CoCreateInstance");
            return false;
        }
```
And then unindent the rest of the block